includes/definitions/ecreso-fm-tx.yaml

```diff
@@ -0,0 +1,28 @@
+os: worldcastsystems
+text: "Ecreso FM Transmitter"
+type: transmitter
+group: ecreso
+icon: worldcastsystems_square
+over:
+  - { graph: device_power, text: Power }
+  - { graph: device_voltage, text: Voltage }
+  - { graph: device_count, text: Count }
+discovery:
+  - sysObjectID:
+      - .1.3.6.1.4.1.8072.3.2.10
+
+discovery_modules:
```

Comment:
Unless you have a need to disable all of these, discovery will work out what's available and only poll that. Please remove this section and the poller_modules one.